detox/src/devices/IosDriver.js

```diff
@@ -71,6 +71,10 @@ class IosDriver extends DeviceDriverBase {
   getPlatform() {
     return 'ios';
   }
+
```

Comment:
Add `log.warn` saying something like "pressBack is an android specific function, make sure you create an android specific test for this scenario"